### PR TITLE
Add a "give a talk" page

### PR DESCRIPTION
**Pull request by tombruijn**
This page describes what expectations there are for a talk and what
format they are. I've added this page to take away some of the questions
some people may have, and to have something we can refer potential
speakers to.

Screen Shot 2022-02-23 at 21 37 36-fullpage

### CODE REVIEW

File: speakers/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title: Give a talk!
+description: "Interested in giving a talk at Amsterdam.rb? Read this page for all the information you need to prepare and what you can expect."
+---
+
+<main class="light-content content-with-shadow">
+  <div class="wrap text-content">
+    <h1>Give A <span class="highlight">Talk</span></h1>
+    <p>
+      We're always looking for new and experienced speakers to give a talk at our meetup.
+      First time speaker are very welcome to send in their talks.
```
Comment (raytalks) [inline speakers/index.html:12]: *speakers

File: speakers/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    </p>
+    <p>
+      If you are interested in giving a talk at one of our future meetups, awesome!, contact us at <a href="mailto:contact@rubynl.org?subject=Giving%20a%20talk%20at%Amsterdam%20Ruby">contact@rubynl.org</a> with a talk title and a short description. You can send in your talk idea months in advance, even before you have starting working on the slides.
```
Comment (raytalks) [inline speakers/index.html:15]: Suggestion: If you are interested in giving a talk at one of our future meetups, awesome! Contact us at ...

Comment (tombruijn) [reply speakers/index.html:15]: I'll add a message that this can be found at the top of the page. I don't want to repeat that content again.

**Comment by emcoding** [comment]
I like it!
One thing though. I myself never felt un-welcome to give a talk by any of you = orga team. My hesitation's more to do with the audience ;-) That's what I was talking with Arno about: I don't feel comfortable talking about Ruby or Rails topics in this group.  I told Arno that I am going to do a practice run at wnb.rb, talking about validations in Rails. Only when I told him, I realized that I don't have any hesitations about giving the talk at wnb.rb. In that group too are people who are far more experienced programmers, and I don't expect them to learn anything new from my talk. That's not a reason not to do the talk. 
So I wonder what is the difference? I suppose it has to do with expectation management. 
What are take aways? Maybe: emphasize that ams.rb strives for having 1 first time speaker and 1 experienced talk for every meetup. So, the audience knows what to expect. 

In the meantime, I'm thinking about a non-technical subject to talk about. :-)

File: speakers/index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    <h3>Topic</h3>
+    <p>
+      At Amsterdam.rb, it's to be expected some talks will be about Ruby. New things in Ruby releases or cool new gems, etc.
+    </p>
+    <p>
+      But while this meetup is called "Amsterdam.rb", not all talks need to be about Ruby. We also accept talks that are not about Ruby, but instead related to software, the act of programming, managing, communication, important issues in our community, etc.
+    </p>
+    <h3>Length</h3>
+    <p>
+      Talks are can be between 15 and 30 minutes. We'll reserve some additional time in the program for Questions and Answers, which we try to limit to 10 minutes.
```
Comment (emcoding) [inline speakers/index.html:31]: are can 
```suggestion
      Talks can be between 15 and 30 minutes. We'll reserve some additional time in the program for Questions and Answers, which we try to limit to 10 minutes.
```

**Comment by tombruijn** [comment]
Thanks for checking the PR @emcoding! I think it would also help to reach out to first time speakers directly and help them with their presentation.
This page is helpful already just so that we've written down a lot of information speakers may want to know before/after agreeing to speak.

**Comment by tombruijn** [comment]
@raytalks I've merged the PR. I think some feedback of you was lost? Please send in a PR or let me know your suggestions later :)

**Pull request merged**